themes/default/content/blog/serverless-architecture-design/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Modern Serverless Architecture Design"
+date: 2023-11-03
+draft: false
+meta_desc: "Learn about the concept of serverless architecture and the different considerations to make when building out your serverless application."
+meta_image: meta.png
+authors:
+    - adora-nwodo
+tags:
+    - serverless
+    - serverless-functions
+    - azure-functions
+    - system-design
+    - system-architecture
+
+---
+
+Serverless architectures have become popular because they make it easier for software developers to focus on application development without the additional operation overhead. With Pulumi, you can deploy and manage your serverless infrastructure and this article shows you how to do this with Azure.
```

Comment:
Seems like we need to rewrite a good portion of this paragraph, since the focus of the article is now architectural/design recommendations and not how to deploy on Azure.